### PR TITLE
CON-2206- Update certificate confirmation page

### PR DESCRIPTION
**Pull request by ben1stone**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml [hunk 1]
```diff
@@ -21,7 +21,7 @@
                         <br /><strong>@Model.GivenNames @Model.FamilyName</strong>
                     </div>
                     <div>
-                         @if (Model.SelectedGrade == CertificateGrade.Fail)
+                        @if (Model.SelectedGrade == CertificateGrade.Fail)
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:24]: The AC's state that the screen will change when it is NOT a fail - I'm not sure that is correct - it should probably change for both a pass and a fail - but the fail case has slightly different text - the implementation seems more correct than the AC's but they should match for testing - I think the AC's should state what happens for a pass and for a fail

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml [hunk 2]
```diff
@@ -35,10 +35,16 @@
                 </div>
             </div>
 
+            <h2 class="govuk-heading-m">What happens next</h2>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:38]: The AC's state the colour is changing - I don't see a change for this unless it's CDN supplied? and I didn't think this was using a CDN?

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml [hunk 3]
```diff
@@ -35,10 +35,16 @@
...
+
+            <p class="govuk-body"> We'll send the apprentice's certificate to the address you entered. You can <a class="govuk-link" asp-controller="CertificateHistory" asp-action="Index">check the status of the certificate</a> in the 'Completed assessments' section of your account.</p>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:40]: Mostly this page uses a localizer to supply the text - I think it should either continue to mostly do so - for the new text or not do that at all and convert back to inline text completely.

My preference would be to convert it to inline text as mostly the rest of the application does not use localizers - as EFSA is only for England...

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml (hunk 1 above)
Comment (cofaulco) [reply src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:24]: AC's updated, should not show new content for Fail grade however should display the new link

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml (hunk 2 above)
Comment (cofaulco) [reply src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:38]: Upon investigation, the new GDS colour has already been implemented.

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml (hunk 3 above)
Comment (cofaulco) [reply src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:40]: As per @ChrisJWoodcock  recommendation, remove Localizer for this View.

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml [hunk 4]
```diff
@@ -35,16 +35,22 @@
...
+                    We'll send the apprentice's certificate to the address you entered. You can <a class="govuk-link" asp-controller="CertificateHistory" asp-action="Index">check the status of the certificate</a> in the 'Completed assessments' section of your account.
+                </p>
+            }
             <h2 class="govuk-heading-m">@Localizer["DoNextHeader"]</h2>
             <ul class="govuk-list">
                 <li><a class="govuk-link" asp-controller="Search" asp-action="Index">@Localizer["DoNextSearch"]</a></li>
                 <li><a class="govuk-link" href="https://www.smartsurvey.co.uk/s/certification-transaction-complete/" target="_blank">@Localizer["DoNextFeedback"]</a></li>
-                <li>or</li>
-                <li><a class="govuk-link" asp-controller="Dashboard" asp-action="Index">Return to account home</a></li>
+                @if (Model.SelectedGrade != CertificateGrade.Fail)
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml:49]: I don't think AC4 required any changes - it was a test scenario to show that the link at the end applied to both the Fail and the Pass - so this is now broken

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]
The removal of the Localizer on this page has not been done

File: src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml (hunk 3 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml]: @shomavg - you haven't done this yet

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**